experiments/build-swiftshader.md

```diff
@@ -0,0 +1,22 @@
+# Building SwiftShader with Coverage Enabled
+
+The SwiftShader CMakeLists.txt file defines the SWIFTSHADER\_EMIT\_COVERAGE flag to generate coverage information. However, it was designed for a very specific toolchain (clang-10 toolchain, CMake option SWIFTSHADER\_LLVM\_VERSION=10.0).
+
+The build at https://swiftshader-review.googlesource.com/c/SwiftShader/+/44728 can be built with coverage information without these restrictions.
+
+## Steps:
+
+1. Clone the [repo](https://swiftshader-review.googlesource.com/c/SwiftShader/+/44728)
+2. Enable the flag SWIFTSHADER\_EMIT\_COVERAGE in CMakeLists.txt
+3. Build as described in the SwiftShader README
+
+# Running Vulkan programs with SwiftShader
+
+In order to generate the complete coverage information for a Vulkan sample, the sample needs to be built with coverage as well. This will create the appropriate .gcno files for the sample. In order to create the .gcda files for both the sample and SwiftShader, we need to link the Vulkan sample with the SwiftShader library when we run the Vulkan samples.
```

Comment:
I suggest removing this. We should not need to build the samples with coverage enabled.